vocabularies/CodeList.json

```diff
@@ -91,6 +91,15 @@
             "@Common.Experimental": true,
             "@Core.Description": "Property contains a Configuration Deprecation Code",
             "@Core.LongDescription": "The Configuration Deprecation Code indicates whether a code list value is valid (deprecation code is empty/space), deprecated (deprecation code `W`), or revoked (deprecation code `E`). "
+        },
+        "DoNotCheckScale": {
+            "$Kind": "Term",
+            "$Type": "Edm.Boolean",
+            "$AppliesTo": [
+                "Property"
+            ],
+            "@Core.Description": "Do not check the allowed number of fractional digits",
+            "@Core.LongDescription": "The annotated property contains a measured quantity, and the property value may have more fractional digits than defined for its unit of measure."
```

Comment:
... quantity, and the user may enter more fractional digits ...

This would switch off the validation of user input with respect to decimals.

Reply:
So is this a UI annotation? It seems to tell the UI "do not complain about too many decimal places, just drop them silently".

If yes, this belongs in the UI vocabulary.

Reply:
Moved to UI